cloudformation-templates/zs_cc_cf_template_simple.yaml

```diff
@@ -418,7 +444,7 @@ Outputs:
     Export:
       Name: !Sub '${AWS::StackName}-MgmtPrivateIP'
   ServiceEni:
-    Description: Zscaler Cloud Connector Instance Service Eni
+    Description: Zscaler Cloud Connector Instance Service Eni for traffic forwarding
```

Comment:
it's technically "ENI" capitalized acryonym

Reply:
Gotcha, I'll change that.